src/qt/bitcoingui.cpp

```diff
@@ -555,6 +574,11 @@ void BitcoinGUI::removeAllWallets()
     setWalletActionsEnabled(false);
     walletFrame->removeAllWallets();
 }
+
+bool BitcoinGUI::multiWallet() const
+{
+    return (WalletSelector->count() > 1);
```

Comment:
Using a UI element to count the number of wallets doesn't seem ideal.

Reply:
I think a better approach is to give QT access to `std::vector<CWalletRef> vpwallets`, and then when the dropdown box (re-)renders itself, it should query that.

That would also avoid passing walletModel instances around like in `rpcConsole->addWallet(walletModel);`.

Reply:
Agree with @Sjors